src/App.js

```diff
@@ -26,9 +26,10 @@ class App extends React.Component {
 
   displaySelectedMovie = (id) => {
     const filteredMovie = this.state.allMovies.filter(movie => movie.id === id)
+    // console.log('line28', filteredMovie[0].id)
     this.setState({
       test: false,
-      SelectedMovie: filteredMovie
+      SelectedMovie: filteredMovie[0].id
```

Comment:
What is the length of this 'filteredMovie' array? Aren't the IDs unique, and shouldn't we only return one movie that matches the ID? It seems a bit hardcoded to me, using the index number here to get that specific movie from 'filteredMovie'.